controllers/fetchers/url_fetcher.go

```diff
@@ -30,7 +31,8 @@ func FetchDashboardFromUrl(dashboard *v1beta1.GrafanaDashboard) ([]byte, error)
 		return nil, err
 	}
 
-	client := client2.NewInstrumentedRoundTripper(fmt.Sprintf("%v/%v", dashboard.Namespace, dashboard.Name), metrics.DashboardUrlRequests, true)
+	tlsConfig := client.BuildInsecureTLSConfiguration()
+	client := client2.NewInstrumentedRoundTripper(fmt.Sprintf("%v/%v", dashboard.Namespace, dashboard.Name), metrics.DashboardUrlRequests, true, tlsConfig)
```

Comment:
This is an interesting case. I'd personally use the same TLS config as the instance if set, and a secure default if not

Reply:
I have made a refactoring to isolate this behavior. However, I don't feel like it is a good idea to use the same certificate than the Grafana to retrieve the value of this URL. This could be a completely different certificate authority.

I have isolated this usecase in the dashboard controller. I think an evolution for this object could be an idea but it is too heavy for this PR.